turtlebot3c-bringup-snap/snap/snapcraft.yaml

```diff
@@ -4,40 +4,40 @@ license: GPL-3.0
 summary: Turtlebot3c core snap
 description: |
   This snap automatically spawn a roscore and the core components for the
-  Turtlebot3 (controller, robot_state_publisher etc).
+  Turtlebot3.
 
-  By default, the snap is configured for the 'waffle_pi' Turtlebot3 model.
-  To select another robot model (waffle or burger) run the following,
+  This snap is meant to run on the Turtlebot3 robot or with the simulation
 
-    snap set turtlebot3c-bringup turtlebot3-model=burger
+  By default the snap runs on the robot.
+  To switch to the simulation
 
-  Similarly, the snap is configured to consider the lidar model LDS-01.
-  To select the other lidar for the robot run the following,
+    snap set turtlebot3c simulation=true
 
-    snap set turtlebot3c-bringup lds-model=LDS-02
+  By default the snap used the lidar LDS-01 robot.
+  To select the other lidar for the robot
 
-  This snap is meant to run on the Turtlebot3 robot,
-  but it can also be used with the simulation.
+    snap set turtlebot3c lds-model=LDS-02
 
-  In order to use it with the simulation, issue the following in the terminal,
+  By default the snap runs the waffle_pi Turtlebot3 model.
```

Comment:
```suggestion
  By default the snap runs the `waffle_pi` Turtlebot3 model.
```

Reply:
should this be changed even tho is in the snapcraft.yaml?

Reply:
apparently, before it was:
```
 By default, the snap is configured for the 'waffle_pi' Turtlebot3 model.
```

Reply:
was it causing an issue.